thirdTask/thirdTask/ViewController.swift

```diff
@@ -1,12 +1,12 @@
 import UIKit
 
-class ViewController: UIViewController, UITableViewDataSource, UITableViewDelegate {
+class ViewController:  UIViewController,  UITableViewDataSource,  UITableViewDelegate {
     
-    @IBOutlet var textDescription: UITextView!
-    @IBOutlet var colorsTable: UITableView!
-    
-    // Define an array of colors and their corresponding names
-    let colors: [(UIColor, String)] = [
+    @IBOutlet var textDescription:  UITextView!
```

Comment:
this should be @IBOutlet var textDescription: UITextView! without a space before : apply this to the whole code.